heartbeat/docs/heartbeat-options.asciidoc

```diff
@@ -467,11 +467,62 @@ Under `check.request`, specify these options:
 *`headers`*:: A dictionary of additional HTTP headers to send.
 *`body`*:: Optional request body content.
 
+Example configuration:
+
+[source,yaml]
+-------------------------------------------------------------------------------
+- type: http
+  #
+  # This monitor POSTs an x-www-form-urlencoded string
```

Comment:
Not a big deal, but I would make the comment introductory text rather than part of the example. Why? Because someday our docs will be translated. Any content that appears within examples will not be translated.

Reply:
Super, I will move.